src/pray.c

```diff
@@ -722,10 +720,6 @@ int godnum;
 		return;
 	}
 
-	if(Inhell && godnum != GOD_MOLOCH && godnum != GOD_LOLTH /*&& !(Race_if(PM_DROW) && (resp_god != A_LAWFUL || !flags.initgend))*/){
```

Comment:
Dropping this makes me nervous? This could change other behavior besides goatmom?